bpf/go_common.h

```diff
@@ -16,27 +16,14 @@
 #include "utils.h"
 #include "bpf_dbg.h"
 #include "http_trace.h"
+#include "ringbuf.h"
 
 char __license[] SEC("license") = "Dual MIT/GPL";
 
 // TODO: make this user-configurable and modify the value from the userspace when
 // loading the maps with the Cilium library
 #define MAX_CONCURRENT_REQUESTS 500
 
-// setting here the following map definitions without pinning them to a global namespace
```

Comment:
I moved this into a common header that we can use across golang and otherlang support.